### PR TITLE
Frontend skeleton the 37

### PR DESCRIPTION
**Pull request by alexandra-plana**
Initiated next.js client
Added base components (Board, DashBoard, Board Section and Card)
Added mock task data
No styling

### CODE REVIEW

File: client/styles/globals.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+html,
+body {
+  padding: 0;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
```
Comment (thaistcosta) [inline client/styles/globals.css:5]: yay ! 🍎

File: client/common/components/BoardSection.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Task from '../types/Task';
+import Card from './Card';
+
+interface BoardInterface {
+  title: String;
```
Comment (thaistcosta) [inline client/common/components/BoardSection.tsx:6]: Is this interface going in the future for the types folder ?

**Review by thaistcosta — APPROVED** [review]

File: client/common/types/Task.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+export default interface Task {
+  id: Number;
+  title: String;
+  description: String;
+  status: String;
+  priority: String;
+  tags: String[]; //Tags[]
+  deadline: String;
+  milestone_id: String;
```
Comment (lexarisa) [inline client/common/types/Task.ts:9]: all of the ids are stored as Int type, or Number in the database.

File: client/common/components/Card.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Task from '../types/Task';
+
+function Card(task: Task) {
+  return (
+    <div>
+      <h3>{task.title}</h3>
+      <p>{task.description}</p>
+      <p>{task.tags}</p>
+    </div>
```
Comment (lexarisa) [inline client/common/components/Card.tsx:10]: let's add the user assigned to the task as well!

**Review by lexarisa — COMMENTED** [review]
I left some comments :)

Very nice work!

File: client/common/components/BoardSection.tsx (hunk 2 above)
Comment (alexandra-plana) [reply client/common/components/BoardSection.tsx:6]: I'll move them all to interfaces so the code is cleaner :) The types for the props (you'll see that in the next commit) have to stay on the same page though.

File: client/common/types/Task.ts (hunk 3 above)
Comment (alexandra-plana) [reply client/common/types/Task.ts:9]: Thank you, i'lll change that right away

**Review by lexarisa — APPROVED** [review]

**Pull request merged**